test/post-911-gib-status/actions/post-911-gib-status.unit.spec.js

```diff
@@ -0,0 +1,171 @@
+import { expect } from 'chai';
+import sinon from 'sinon';
+import { getEnrollmentData } from '../../../src/js/post-911-gib-status/actions/post-911-gib-status';
+import {
+  BACKEND_SERVICE_ERROR,
+  GET_ENROLLMENT_DATA_FAILURE,
+  GET_ENROLLMENT_DATA_SUCCESS
+} from '../../../src/js/post-911-gib-status/utils/constants';
+
+let oldFetch;
+let oldSessionStorage;
+let oldWindow;
+const setup = () => {
+  oldSessionStorage = global.sessionStorage;
+  oldFetch = global.fetch;
+  oldWindow = global.window;
+  global.sessionStorage = { userToken: '123abc' };
+  global.fetch = sinon.stub();
+  global.fetch.returns(Promise.resolve({
+    headers: { get: () => 'application/json' },
+    ok: true,
+    json: () => Promise.resolve({})
+  }));
+  global.window.dataLayer = [];
+  global.window.URL = {
+    createObjectURL: () => { },
+    revokeObjectURL: () => { }
+  };
+};
+
+const teardown = () => {
+  global.sessionStorage = oldSessionStorage;
+  global.fetch = oldFetch;
+  global.window = oldWindow;
+};
+
+describe('getEnrollmentData', () => {
+  const successResponse = {
+    meta: {
+      status: 'OK'
+    },
+    data: {
```

Comment:
Where did you get this data (structure) from?

Reply:
Good question - copy / pasted from the Swagger for this endpoint